docs/markdown/Commands.md

```diff
@@ -0,0 +1,156 @@
+# Command-line commands
+
+There are two different ways of invoking Meson. First, you can run it directly
+from the source tree with the command `/path/to/source/meson.py`. Meson may
+also be installed in which case the command is simply `meson`. In this manual
+we only use the latter format for simplicity.
+
+Meson is invoked using the following syntax:
+`meson [COMMAND] [COMMAND_OPTIONS]`
+
+This section describes all available commands and some of their notable options.
```

Comment:
I think for newbies, it might be best to list the most common commands and also the order in which you'd use them (generally). F.ex.:

```suggestion
This section describes all available commands and some of their notable options. The most common workflow is to run [`setup`](#setup), followed by [`compile`](#compile), and then [`install`](#install).
```

Reply:
We already have `Overview` though that describes the usual workflow.

Reply:
Yes, but we still have to cater for people that show up to this page as their first piece of documentation from Google. Duplication is good sometimes.

Reply:
Fixed